google/provider.go

```diff
@@ -232,3 +232,10 @@ func validateCredentials(v interface{}, k string) (warnings []string, errors []e
 
 	return
 }
+
+func lockedCall(lockKey string, f func() error) error {
```

Comment:
utils.go might be a better place for this

Reply:
sg